.github/workflows/nightly.yml

```diff
@@ -26,77 +32,181 @@ jobs:
         continue-on-error: true
 
       - name: Build binaries
-        run: cargo build --release --bin safenode --bin safe --bin faucet
+        run: cargo build --release --features local --bin safenode --bin autonomi
         timeout-minutes: 30
 
       - name: Start a local network
         uses: maidsafe/sn-local-testnet-action@main
         with:
           action: start
-          interval: 2000
+          enable-evm-testnet: true
           node-path: target/release/safenode
-          faucet-path: target/release/faucet
           platform: ${{ matrix.os }}
           build: true
 
-      - name: Check contact peer
+      - name: Check if SAFE_PEERS and EVM_NETWORK are set
         shell: bash
-        run: echo "Peer is $SAFE_PEERS"
+        run: |
+          if [[ -z "$SAFE_PEERS" ]]; then
+              echo "The SAFE_PEERS variable has not been set"
+              exit 1
+          elif [[ -z "$EVM_NETWORK" ]]; then
+              echo "The EVM_NETWORK variable has not been set"
+              exit 1
+          else
+              echo "SAFE_PEERS has been set to $SAFE_PEERS"
+              echo "EVM_NETWORK has been set to $EVM_NETWORK"
+          fi
 
       # only these unit tests require a network, the rest are run above in unit test section
-      - name: Run sn_client --tests
-        run: cargo test --package sn_client --release --tests
+      - name: Run autonomi --tests
+        run: cargo test --package autonomi --tests -- --nocapture
         env:
-          SN_LOG: "all"
+          SN_LOG: "v"
           # only set the target dir for windows to bypass the linker issue.
           # happens if we build the node manager via testnet action
           CARGO_TARGET_DIR: ${{ matrix.os == 'windows-latest' && './test-target' || '.' }}
         timeout-minutes: 15
 
-      - name: Create and fund a wallet to pay for files storage
+
+      # FIXME: do this in a generic way for localtestnets
+      - name: export default secret key
+        if: matrix.os != 'windows-latest'
+        run: echo "SECRET_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80" >> $GITHUB_ENV
```

Comment:
This secret keys should be a repo secret maybe?

Reply:
Hey @mazzi, this is just the local evm SECRET KEY, we don't mind exposing it.
The solution would be probably to let the CLI use it without configuring things.